src/components/templates/Jazzbar/JazzTab/JazzTab.tsx

```diff
@@ -141,12 +141,11 @@ const Jazz: React.FC<JazzProps> = ({ setUserList, venue }) => {
         </div>
       )}
 
-      {/* @debt Move the logic of Back button into a separate reusable hook/component */}
```

Comment:
This is probably still relevant I believe, as it relates to not having the back button defined explicitly here at all, but having it defined somewhere 'higher up' like `TemplateWrapper`/similar.

Reply:
Actually, it is being used here. Some venue templates have their own back button. That's why I'm checking for these specific venue templates in the `TemplateWrapper` and not rendering it in there.